src/plugins/Sharesite/webui/EditToadlet.java

```diff
@@ -56,13 +59,14 @@ public void handleMethodGET(URI uri, HTTPRequest req, ToadletContext ctx) throws
 
 		PageNode pageNode = pageMaker.getPageNode(l10n.getString("Sharesite.Menu.Name"), ctx);
 		HTMLNode editForm = pr.addFormChild(pageNode.content,"/Sharesite/Edit/" + siteId, "editForm");
-		addNodes(editForm,c.getName(),c.getDescription(),c.getText(),c.getCSS(), c.getActivelinkUri(),c.getRequestSSK(),c.getInsertSSK());
+		String iHour = Integer.toString(c.getInsertHour());
```

Comment:
getInsertHour never returns null: if insertHour is null, getInsertHour returns -1. Which is not what I want, but is in there for backwards compatibility (to avoid complaints from users that their sharesites suddenly do not insert instantly anymore). Can’t avoid that since there already are several users …